packages/cli/src/zosfiles/edit/Edit.utils.ts

```diff
@@ -0,0 +1,381 @@
+/*
+* This program and the accompanying materials are made available under the terms of the
+* Eclipse Public License v2.0 which accompanies this distribution, and is available at
+* https://www.eclipse.org/legal/epl-v20.html
+*
+* SPDX-License-Identifier: EPL-2.0
+*
+* Copyright Contributors to the Zowe Project.
+*
+*/
+
+import { Download, Upload, IZosFilesResponse, IDownloadOptions, IUploadOptions } from "@zowe/zos-files-for-zowe-sdk";
+import { AbstractSession, IHandlerParameters, ImperativeError, ProcessUtils, GuiResult,
+    TextUtils, IDiffNameOptions, CliUtils } from "@zowe/imperative";
+import { CompareBaseHelper } from "../compare/CompareBaseHelper";
+import { existsSync, unlinkSync } from "fs";
+import { tmpdir } from "os";
+import path = require("path");
```

Comment:
Please update to use newer import style with `import` instead of `require`: `import * as path from "path"`